file_paths.md

```diff
@@ -163,160 +163,201 @@ The package will create the following services `puppet`, `mcollective`,
+            lib                                   # :libdir
+            facts                                 # used to generate :factpath
+            reports                               # :reportdir
+            server_data                           # :server_datadir
+            state                                 # :statedir
+            yaml                                  # :yamldir
+
+        etc *                                     # :confdir                     same
+            auth.conf *                           # :rest_authconfig
+            autosign.conf                         # :autosign
+            binder_config.yaml                    # :binder_config
+            csr_attributes.yaml                   # :csr_attributes
+            custom_trusted_oid_mapping.yaml       # :trusted_oid_mapping_file
+            device.conf                           # :deviceconfig
+            fileserver.conf                       # :fileserverconfig
+            puppet.conf *                         # :config
+            routes.yaml                           # :route_file
+            ssl                                   # :ssldir
         var *
-            log                               # :logdir                      same
-                puppet.log                    # not enabled by default
-            run                               # :rundir                      same
-                agent.pid                     # :pidfile
+            log *                                 # :logdir                      same
+                puppet.log                        # not enabled by default
+            run *                                 # :rundir                      same
+                agent.pid                         # :pidfile
 
     C:\ProgramData\PuppetLabs\pxp-agent *
         etc *
-            pxp-agent.conf                    # pxp-agent configuration file
-            modules *                         # stores configuration files for pxp-agent modules
-                pxp-module-puppet.conf        # configuration file of the pxp module puppet (optionally, to override puppet.bat location)
+            pxp-agent.conf                        # pxp-agent configuration file
+            modules *                             # stores configuration files for pxp-agent modules
+                pxp-module-puppet.conf            # configuration file of the pxp module puppet (optionally, to override puppet.bat location)
         var *
             log *
-                pxp-agent.log                 # enabled by default
-            spool *                           # directory containing results of pxp-agent modules
+                pxp-agent.log                     # enabled by default
             run *
+            spool *                               # directory containing results of pxp-agent modules
+
+    C:\Program Files\Puppet Labs\Puppet\bin *     # symlink targets of puppet related binaries
```

Comment:
No - waiting for @McdonaldSeanp to finish [RE-6892](https://tickets.puppetlabs.com/browse/RE-6892) and will then update.